README.md

```diff
@@ -143,6 +143,41 @@ following scripts are provided:
   found, "openqa-investigate". Compare to section
   "Combine auto-review and openqa-investigate"
 
+### openqa-investigate comments
```

Comment:
As there is already the section in https://github.com/os-autoinst/scripts/pull/265/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R94 I would make that

```suggestion
### More details and examples about openqa-investigate comments
```